ci-operator/step-registry/cucushift/installer/rehearse/ibmcloud/ipi/dedicatedhost/static/deprovision/cucushift-installer-rehearse-ibmcloud-ipi-dedicatedhost-static-deprovision-chain.yaml

```diff
@@ -0,0 +1,10 @@
+chain:
+  as: cucushift-installer-rehearse-ibmcloud-ipi-dedicatedhost-static-deprovision
+  steps:
+  - chain: gather-network
+  - chain: gather-core-dump
+  - ref: gather-ibmcloud-resource
+  - chain: ipi-deprovision
```

Comment:
You can reference `cucushift-installer-rehearse-ibmcloud-ipi-deprovision` instead of the above steps.